README.md

```diff
@@ -204,6 +205,12 @@ Unfortunately, it's not possible to get the image of stickers sent by users. You
 
 This feature is not supported on the Messaging API. Users can only play videos that are sent from a bot, they cannot download or forward the videos to other users.
 
+### Why doesn't my audio, video, or image message display properly on Android devices?
+
+The Android LINE App does not currently support Server Name Indication (SNI) on devices that are using versions of Android older than 6.0. As a result, audio, video, and image messages that contain media that is hosted from a server that uses SNI may not be displayed properly on these Android devices. Our team is planning to fix this in a future release.
```

Comment:
@mrmaakun @kenneth-lau 
According to Okamoto-san's comment on LINEAND-15366, SNI support on Android 6+ is NOT intentional. So I think mentioning exact OS version has some risks